### PR TITLE
feat: adds possibility to use ressource-icons

### PR DESCRIPTION
**Pull request by NLueg**
Hi there!
I wanted to add some Icons and actually wasn't able to transform my SVG-Icon to an encoded bitmap.
That's why I opened the API a bit to add an Icon by a resource name (e.g. ic_launcher_background).

I'm looking forward to your review! 😄

### CODE REVIEW

File: android/src/main/java/nepheus/capacitor/androidshortcuts/AndroidShortcuts.java [hunk 1]
```diff
@@ -71,21 +82,55 @@ public void addPinned(Bridge bridge, String id, String shortLabel, String longLa
...
         shortcutManager.requestPinShortcut(shortcut, null);
     }
 
+    /**
+     * Generates an Icon based on the given parameter
+     * @param shortcutIcon parameter of the icon that should be set
+     * @return
+     */
+    @RequiresApi(api = Build.VERSION_CODES.M)
+    public Icon generateIcon(
```
Comment (NLueg) [inline android/src/main/java/nepheus/capacitor/androidshortcuts/AndroidShortcuts.java:96]: Here is the new method to generate an Icon based on the new API.

**Comment by NePheus** [comment]
https://github.com/NePheus/capacitor-android-shortcuts/wiki/Icon-examples
I made two examples in the wiki for including icons. One for transforming svgs and one for assets in the js project.

I will have a closer look to this PR the next days. Its a great idea to have more opportunities for the icon.

**Comment by NLueg** [comment]
Thank you for your fast response!
I actually don't saw the wiki, thank you very much for pointing it out.

**Comment by NLueg** [comment]
Thanks for your feedback and sorry for the late response!
I will try to also use an enum in the Java code. Also, I will update the documentation.
The PR will be updated today EOB 😄

**Comment by piotr-cz** [comment]
The wiki > [Icon examples](https://github.com/NePheus/capacitor-android-shortcuts/wiki/Icon-examples) still uses the `iconBitmap` format

**Comment by NePheus** [comment]
Updated! Thanks for pointing this out.

**Pull request merged**